### PR TITLE
Update README to Prepare for GA

### PR DESCRIPTION
**Pull request by jaxdesmarais**
### Summary of changes

- Update README to prepare for GA

### Checklist

- ~[ ] Added a changelog entry~

### Authors

- @jaxdesmarais

### CODE REVIEW

**Comment by jaxdesmarais** [comment]
We can wait to merge this until after the release or can merge day of before we kick off the release.

**Review by scannillo — APPROVED** [review]
Do we want to remove the V5 migration guide too from `master`?

**Review by sshropshire — APPROVED** [review]

File: README.md
Comment (scannillo) [reply README.md:11]: Great idea 👏

**Pull request merged**
